src/scripts/components/Chart.jsx

```diff
@@ -30,7 +30,7 @@ class Chart extends React.Component {
   componentDidMount() {
```

Comment:
I'm thinking we should remove the `hasChartData` from `componentDidMount` since the data might be fetching on initial render. I think you can remove it from state as well and just check if you have data in the `render` function.